src/tom-select.ts

```diff
@@ -89,7 +90,7 @@ export default class TomSelect extends MicroPlugin(MicroEvent){
 	public userOptions				: {[key:string]:boolean} = {};
 	public items					: string[] = [];
 
-	private refreshTimeout			: null|ReturnType<typeof setTimeout> = null;
+	private refreshTimeout			: null|number = null;
```

Comment:
`ReturnType<typeof setTimeout>` was transformed to `NodeJS.Timeout` during the build process. This resulted in an error when I tried using this new version in my project:

```
../../node_modules/tom-select/dist/esm/utils.d.ts:27:68 - error TS2503: Cannot find namespace 'NodeJS'.

27 export declare const timeout: (fn: () => void, timeout: number) => NodeJS.Timeout | null;
                                                                      ~~~~~~


Found 1 error.
```

Since this should always run in the browser, it's safe to use `number` here. See corresponding change in `src/utils.ts` from `setTimeout` to `window.setTimeout`, and the change in `onInput` below from `clearTimeout` to `window.clearTimeout`.